src/i2c.rs

```diff
@@ -126,12 +109,16 @@ impl<I, SDA, SCL> I2c<I, SDA, SCL>
 
         // Configure for "fast mode" (400 KHz)
         i2c.timingr.write(|w| {
-            w
-                .presc().bits(presc)
-                .scll().bits(scll)
-                .sclh().bits(sclh)
-                .sdadel().bits(sdadel)
-                .scldel().bits(scldel)
+            w.presc()
+                .bits(presc)
+                .scll()
+                .bits(scll)
+                .sclh()
+                .bits(sclh)
+                .sdadel()
+                .bits(sdadel)
+                .scldel()
+                .bits(scldel)
```

Comment:
This is an example of code that has become much less readable as a result of running `rustfmt` (see my top-level comment in the pull request). Arguably there are other kinds of examples too, but it's really obvious and egregious with these svd2rust-generated APIs.

Reply:
Great point, I had not considered this and I'm inclined to agree that readability is far more important than consistency. I'll start following the readable format moving forward.